src/Views/Form/Themes/Sequoia/Actions.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Give\Views\Form\Themes\Sequoia;
+
+use Give_Donate_Form;
+use function Give\Helpers\Form\Utils\isViewingForm;
+
+
+/**
+ * Class Actions
+ *
+ * @since 2.7.0
+ * @package Give\Form\Themes\Sequoia
+ */
+class Actions {
```

Comment:
I would really like to explore as to how we can streamline what this file is doing. This seems to be the most tedious part of setting up a theme at this point. There are a ton of hooks, some of which are just for injecting closing tags. Are any of these hooks doing something that would be useful to every theme?

Reply:
@JasonTheAdams No, this file does not any common form theme code. These hooks added to modify form DOM structure in favor of javascript to achieve the design.

Reply:
That's exactly my point. Can we simply update the current form structure (or update the structure for all themes), such that all a theme really has to provide is styles and scripts?

Reply:
@JasonTheAdams No, I will not advise doing that because changing because it is form theme responsibility to update form dom structure according to there need.

Reply:
You don't think there are any basic structures that can be applied? Take, for example, the fact that we're breaking the form up into multiple steps. That's really just CSS/JS. Having each a structure that easily supports this would seem useful to me, and the legacy form simply wouldn't do anything with these in terms of styling.

Reply:
As per discussion, I will create a different issue for this.